src/api/app/assets/stylesheets/webui2/list-group-item-callout.scss

```diff
@@ -0,0 +1,32 @@
+.list-group-item-callout-warning {
+    border-left-color: $main_yellow;
+}
+
+.list-group-item-callout-success {
+    border-left-color: color("green");
+}
+
+.list-group-item-callout-danger {
+    border-left-color: color("red");
+}
+
+.list-group-item-callout-info {
+    border-left-color: #ffffff00;
+}
+
+.list-group-item-callout {
+    border-right: none;
+    border-left-width: .5rem;
+}
+
+.list-group-item:first-child.list-group-item-callout {
+    border-top-left-radius: 0rem;
+}
+
+.list-group-item:last-child.list-group-item-callout {
+    border-bottom: none;
+}
+
+.list-group .list-group-item-integrated:last-child {
+    border-bottom: none;
+}
```

Comment:
I like the design! :fireworks: 

There are [border classes](https://getbootstrap.com/docs/4.1/utilities/borders/) for all of this.

Reply:
I know... but then I have to code for adding that simple classes (eg mb-0 or pb-0) each time a have a list callout. Instead using CSS I don't have to calculate which one is the last item in the list.

Reply:
> each time a have a list callout

You think this will be used more than one time? For what?

Reply:
This is used in every single list of items inside a card that is filling the card-body to not have double borders.

Reply:
Ok